api/provider/aws/test_aws/task_read_test.go

```diff
@@ -110,3 +110,96 @@ func TestTaskRead(t *testing.T) {
 
 	assert.Equal(t, expected, result)
 }
+
+func TestTaskRead_CannotPullContainer(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockAWS := awsc.NewMockClient(ctrl)
+	tagStore := tag.NewMemoryStore()
+	mockConfig := mock_config.NewMockAPIConfig(ctrl)
+
+	mockConfig.EXPECT().Instance().Return("test").AnyTimes()
+
+	tags := models.Tags{
+		{
+			EntityID:   "tsk_id",
+			EntityType: "task",
+			Key:        "name",
+			Value:      "tsk_name",
+		},
+		{
```

Comment:
I believe the `arn` tag is the only tag required to do the proper lookup; all others should be removed.

Reply:
`environment_id` is also required; I thought about removing the other tags but your original comment was ambiguous